piecedirectory/doctor.go

```diff
@@ -7,36 +7,37 @@ import (
 	"math/rand"
 	"time"
 
-	"github.com/filecoin-project/boost/piecedirectory/types"
+	"github.com/filecoin-project/boost/db"
+	"github.com/filecoin-project/boost/sectorstatemgr"
+	bdclient "github.com/filecoin-project/boostd-data/client"
+	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/ipfs/go-cid"
 	logging "github.com/ipfs/go-log/v2"
 )
 
 var doclog = logging.Logger("piecedoc")
 
-type SealingApi interface {
```

Comment:
`Piece Doctor` no longer accesses directly APIs from Lotus, and relies on information from the `Sector State Manager`.